.dockerignore

```diff
@@ -0,0 +1,3 @@
+.env
+.env.template
+.env.test.env
```

Comment:
These cover the most important files to avoid copying (particularly .env, as we don't want to bake secrets into our image) but there are various things we could add here:
* Anything in our `.gitignore`, for example the `.venv` folder storing your local dependencies doesn't want to be copied in because we want our image to reproducibly create that itself
* Anything not required by our app - the Dockerfile itself potentially, documentation etc.